app/models/fetch/summaries.py

```diff
@@ -0,0 +1,157 @@
+import datetime
+import logging
+from typing import List
+from statistics import mean
+from numpy import percentile
+from geoalchemy2.shape import to_shape
+from scipy.interpolate import griddata
+import app.db.database
+from app.db.crud import get_coordinates_predictions
+from app.db.models import ModelRunGridSubsetPrediction, PredictionModelGridSubset, PredictionModel
+from app.schemas import (
+    WeatherForecastModelSummary, WeatherForecastModelSummaryValues, WeatherStation, WeatherForecastModel)
+from app.models import ModelEnum
+from app.wildfire_one import get_stations_by_codes
+from app.models.fetch import extract_stations_in_polygon
+
+
+logger = logging.getLogger(__name__)
+
+
+KEYS = ('tmp_tgl_2', 'rh_tgl_2')
+
+
+def _build_query(stations: List[WeatherStation], model: ModelEnum):
+    """
+    """
```

Comment:
sneaky way to get pylint to stop complaining about missing docstrings? 😉

Reply:
An honest oversight - I promise!